### DISCUSSION TITLE
Github issue template

### DISCUSSION THREAD

**Discussion by slumbering** [dagger/dagger#1546] [Ideas]
cc @dagger/team 

I think we should add a very simple Github issue template. Therefore, we can gather minimum information if we need to reproduce bugs. This is something we've already talked about with @gerhard @grouville and @TomChv.
Here is a simple example (TBD):

---

Issue tracker is **ONLY** used for reporting bugs.

## Expected Behavior
 Tell us what should happen

## Current Behavior
 Tell us what happens instead of the expected behavior 

## Steps to Reproduce

1.
2.
3.
4.

## Context (Environment)

  - Dagger Version:
  - Operating System and version:

**Comment by gerhard** [comment]
This is a great idea! I am keen on us getting familiar with this documentation: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates.

This is a great follow-up: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

As for examples, this looks helpful: https://github.com/stevemao/github-issue-templates

We may want to add this new issue template to https://github.com/dagger/.github so that the universe repo will benefit from it too. Alternatively, adding it directly to this repository may be a good enough first step.

**Reply by slumbering** [reply]
> As for examples, this looks helpful: https://github.com/stevemao/github-issue-templates

Thanks to point this link out. This is actually where I get my simple example ;).
